libtransmission/peer-mgr.c

```diff
@@ -79,7 +79,10 @@ enum
     /* */
     NO_BLOCKS_CANCEL_HISTORY = 120,
     /* */
-    CANCEL_HISTORY_SEC = 60
+    CANCEL_HISTORY_SEC = 60,
+    /* UPLOAD/DOWNLOAD RATE SMOOTHING, 1 = no smoothing, upwards increases smoothing */
+    SMOOTHING_RATE_UP = (5 / 4),
+    SMOOTHING_RATE_DOWN = (3 / 2)
```

Comment:
Both up and down are smoothed to 1 because of these lines: https://godbolt.org/z/Gj78a4Yrz

You probably want to move these to _after_ the enum and have something like:

```
static constexpr double SmoothingRateUp = 5 / 4.0;
static constexpr double SmoothingRateDown = 3 / 2.0;
```